include/superior_mysqlpp/prepared_statements/prepared_statement_base.hpp

```diff
@@ -192,45 +254,71 @@ namespace SuperiorMySqlpp
                 return ok;
             }
 
-            /**
-             * TODO: Statement.fetchColumn is void and can throw
-             * -> do we need bool as a return value here?
-             * @return
-             */
             bool fetchColumn()
             {
                 this->statement.fetchColumn();
                 engageNullables();
                 return true;
             }
 
+        public:
+            /**
+             * Allows sending data for long enough parameters in multiple chunks (through successive calls).
+             * Parameter must have Text or Blob field type.
+             * @param paramNumber Index of given parameter.
+             * @param data C style string.
+             * @param length Lenght of  data in bytes.
+             * @return void
+             * !!!!! Should return void, this form is misleading.
+             */
             auto sendLongData(unsigned int paramNumber, const char* data, unsigned long length)
             {
                 return this->statement.sendLongData(paramNumber, data, length);
             }
 
+            /**
+             * Allows sending data for long enough parameters in multiple chunks (through successive calls).
+             * Parameter must have Text or Blob field type.
+             * @param paramNumber Index of given parameter.
+             * @param data String containing the new data.
+             * @param length Lenght of data in bytes.
+             * @return void
+             * !!!!! Should return void, this form is misleading.
+             */
             auto sendLongData(unsigned int paramNumber, const std::string& data)
             {
                 return this->statement.sendLongData(paramNumber, data);
             }
 
+            /**
+             * Sets how many rows shall be prefetched in single fetch call.
```

Comment:
Does it have meaning for both *use* and *store* results?

Reply:
After looking harder, this one seems problematic. According to https://dev.mysql.com/doc/refman/5.7/en/mysql-stmt-attr-set.html, it does: Number of rows to fetch from server at a time _when using a cursor_.  MariaDB does not mention that at all. We don't even have required method (outside DBDriver) to create cursors.

For now, I'll only add commentary about this.